### DISCUSSION TITLE
How to pass a token to my http API job

### DISCUSSION THREAD

**Discussion by gclsoft** [apache/dolphinscheduler#13397] [General]
The token of my java application API needs a token, expired every ten minutes, how to pass it to the api? I was using xxljob, it has java SDK, so it's easy to just call the java service, is it available in Ds?

**Comment by ghost** [comment]
Hello sir ,
I am new to open source contribution.
I already know java , my tech stacks & tools includes C, C++ , Python , Java, JavaScript , HTML , CSS , SQL , Bootstrap, ReactJS, ExpressJS, NodeJS & Git . I need a little help from your side to contribute to these amazing projects.
